enterprise/app/jobs/sla/process_account_applied_slas_job.rb

```diff
@@ -2,7 +2,10 @@ class Sla::ProcessAccountAppliedSlasJob < ApplicationJob
   queue_as :medium
 
   def perform(account)
-    account.applied_slas.where(sla_status: 'active').each do |applied_sla|
+    account.applied_slas.where.not(sla_status: 'hit').each do |applied_sla|
+      # only proccess applied sla if the conversation is not resolved
+      next if applied_sla.conversation.resolved?
```

Comment:
@sojan-official Yes, since we will be evaluating all three conditions every time. Technically, we will miss out on resolution miss if it's missed by a small margin(scheduled job delay).

Current setup --> https://github.com/chatwoot/chatwoot/blob/1303469087d80706b732e8f4135c138bf7a913ac/spec/enterprise/services/sla/evaluate_applied_sla_service_spec.rb#L94